### PR TITLE
Better copy and naming

### PR DESCRIPTION
**Pull request by jancurn**

### CODE REVIEW

File: loader_hub/apify/actor/base.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
 from llama_index.readers.schema.base import Document
 
 
-class ApifyCallActor(BaseReader):
+class ApifyActor(BaseReader):
     """Apify call Actor reader.
```
Comment (fnesveda) [inline loader_hub/apify/actor/base.py]:
```suggestion
    """Apify Actor reader.
```

File: loader_hub/apify/actor/base.py [hunk 2]
```diff
@@ -6,7 +6,7 @@
...
     It calls an Actor on Apify and reads data when it finihes.
```
Comment (fnesveda) [inline loader_hub/apify/actor/base.py]:
```suggestion
    It calls an Actor on Apify and reads its output when it finihes.
```

File: loader_hub/apify/dataset/README.md [hunk 3]
```diff
@@ -1,20 +1,31 @@
 # Apify Dataset Loader
 
-Apify is a cloud platform for web scraping and data extraction,
+[Apify](https://apify.com/) is a cloud platform for web scraping and data extraction,
 which provides an [ecosystem](https://apify.com/store) of more than a thousand
 ready-made apps called *Actors* for various scraping, crawling, and extraction use cases.
 
-This loader loads documents from an [Apify dataset](https://docs.apify.com/platform/storage/dataset).
+This loader loads documents from an existing [Apify dataset](https://docs.apify.com/platform/storage/dataset).
```
Comment (fnesveda) [inline loader_hub/apify/dataset/README.md:7]: We could crosslink the `ApifyActor` loader here, to increase its discoverability

File: loader_hub/apify/actor/README.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+In this example, we’ll use the [Website Content Crawler](https://apify.com/apify/website-content-crawler) Actor,
+which can deeply crawl websites such as documentation, knowledge bases, help centers,
+or blogs, and extract text content from the web pages. 
+The extracted text then can be fed to a vector index in order to answer questions from it.
+
+To use this loader, you need to have a (free) Apify account 
+and provide your [Apify API token](https://console.apify.com/account/integrations) as the `APIFY_API_TOKEN`
+environment variable. Note that the loader requires
+the [apify-client](https://pypi.org/project/apify-client/) PyPI package.
```
Comment (fnesveda) [inline loader_hub/apify/actor/README.md]: The `apify-client` is installed on its own apparently: https://github.com/emptycrown/llama-hub#step-1-create-a-new-directory

File: loader_hub/apify/actor/README.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+    return Document(
+        item.get('text'),
+        extra_info={
+            "url": item.get('url'),
+        },
+    )
+
+ApifyActor = download_loader('ApifyActor')
+
+reader = ApifyActor(os.environ.get('APIFY_API_TOKEN'))
```
Comment (fnesveda) [inline loader_hub/apify/actor/README.md]: Maybe we don't have to complicate things with the environment variables, and create the loader with the token just as string?
```suggestion
reader = ApifyActor('MY_APIFY_API_TOKEN')
```

File: loader_hub/apify/actor/README.md [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+from llama_index import download_loader
+from llama_index.readers.schema.base import Document
+import os
+
+# Converts a single record from the Actor's resulting dataset to the LlamaIndex format
+def tranform_dataset_item(item):
+    return Document(
+        item.get('text'),
+        extra_info={
+            "url": item.get('url'),
```
Comment (fnesveda) [inline loader_hub/apify/actor/README.md]: It's weird how we mix single and double quotes everywhere, we should choose one.

**Review by fnesveda — CHANGES_REQUESTED** [review]
Added a few comments, some of them apply to both loaders but I wrote them to just one for brevity

File: loader_hub/apify/dataset/README.md (hunk 3 above)
Comment (jancurn) [reply loader_hub/apify/dataset/README.md:7]: Good point, will do

File: loader_hub/apify/actor/README.md (hunk 6 above)
Comment (jancurn) [reply loader_hub/apify/actor/README.md]: I agree, so in Python you can use either?

**Review by fnesveda — APPROVED** [review]

**Pull request merged**